action.yml

```diff
@@ -216,25 +219,27 @@ runs:
             echo "git diff found modified files when it should not have:"
             echo $GIT_DIFF
             echo "sha1sum of files that are different:"
-            COPIED_COUNT=0
             for FILEPATH in $(git diff --cached --name-only); do
               if [[ -f $FILEPATH ]]; then
                 sha1sum $FILEPATH
-                # Only copy if the file is less than 10 megabytes to prevent malicous behaviour
-                # Only copy a max of 10 files, also to prevent malicious behaviour
-                MEGABYTES=$(ls -l --b=M "$DIST_DIR/$FILEPATH" | cut -d " " -f5 | sed "s/M//")
-                if (($MEGABYTES >= 10)); then
-                  echo "File $FILEPATH is larger than 10 megabytes, not copying"
-                elif (($COPIED_COUNT >= 10)); then
-                  echo "More than 10 files have been copied, not copying $FILEPATH"
-                else
-                  cp "$DIST_DIR/$FILEPATH" artifacts
-                  COPIED_COUNT=$((COPIED_COUNT+1))
-                fi
               fi
             done
-            echo "Files that are different have been copied to artifacts directory"
-            exit 1
+            # silverstripe/silverstripe-admin CMS 4 has a known issue where dist files are
+            # sometimes generated differenty from the same source code on subsequent builds.
+            # This doesn't happen in CMS 5, so presumably it's something to do with and
+            # older version of webpack
+            DO_EXIT="true"
+            if [[ $GITHUB_REPOSITORY == "silverstripe/silverstripe-admin" ]]; then
```

Comment:
admin isn't the only repo this runs on - I suspect it's only the one we saw it on most because it's the one that got the most action. If we're only doing this for admin, I don't see any value in it since we're not accepting random PRs anymore and it isn't being run on a cron.

The only time we'll get a false negative now is when running CI for a security release, which is unlikely to happen in the first place but even if it does happen it may need to be run for other repos where this logic will get hit.

We should either ignore it for ALL CMS for stuff (I think we have the installer version in here somewhere, don't we?) or just don't bother doing this at all.

Reply:
Someone else using the CI may have looser restrictions then we do. I think we should continue to limit this to admin since there's really no reason to loosen it further.

Reply:
Removed